src/control/events/ras.go

```diff
@@ -165,6 +179,8 @@ func New(evt *RASEvent) *RASEvent {
 	if evt.Severity == RASSeverityUnknown {
 		evt.Severity = RASSeverityInfo
 	}
+	evt.forwarded = atm.NewBool(false)
+	evt.forwardable = atm.NewBool(true)
```

Comment:
Why not `evt.forwarded.SetFalse()` and `evt.forwardable.SetTrue()`?

Every one of these `NewBool()` calls is an unsynchronized write to a variable. The zero value of an atm.Bool is already usable and set to False.

Looking at the atm API, I do see that there's a gap in that we have a `Load()` but not a `Store()`. I'll fix that.

Reply:
I wasn't clear on the uninitialized state of the variable, I will make those changes. I did notice the lack of Store() , thanks.